src/main/resources/plugin.yml

```diff
@@ -1,8 +1,8 @@
 name: craftableinvframes
-version: ${project.version}
+version: 2.2.2.2
```

Comment:
${project.version}

Reply:
Yes, but i didn't use actions, and I wanna check my version